go.mod

```diff
@@ -270,7 +270,6 @@ require (
 replace (
 	github.com/mdlayher/arp => github.com/kubeovn/arp v0.0.0-20240218024213-d9612a263f68
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.1
-	github.com/ovn-org/libovsdb => github.com/kubeovn/libovsdb v0.0.0-20240218023647-f0bc3ce57fcd
```

Comment:
We replaced the libovsdb with a forked one that already contains `WithInactivityCheck`

Reply:
fix